app/stac_api/admin.py

```diff
@@ -271,6 +272,106 @@ def save_model(self, request, obj, form, change):
         return super().save_model(request, obj, form, change)
 
 
+@admin.register(CollectionAsset)
+class CollectionAssetAdmin(admin.ModelAdmin):
+
+    class Media:
+        js = ('js/admin/asset_help_search.js',)
+        css = {'all': ('style/hover.css',)}
+
+    autocomplete_fields = ['collection']
+    search_fields = ['name', 'collection__name']
+    readonly_fields = [
+        'collection_name',
+        'href',
+        'checksum_multihash',
+        'created',
+        'updated',
+        'etag',
+        'update_interval'
+    ]
+    list_display = ['name', 'collection_name', 'collection_published']
+    fieldsets = (
+        (None, {
+            'fields': ('name', 'collection', 'created', 'updated', 'etag')
+        }),
+        (
+            'File', {
+                'fields': ('file', 'media_type', 'href', 'checksum_multihash', 'update_interval')
+            }
+        ),
+        ('Description', {
+            'fields': ('title', 'description', 'roles')
+        }),
+        ('Attributes', {
+            'fields': ('eo_gsd', 'proj_epsg', 'geoadmin_variant', 'geoadmin_lang')
```

Comment:
I think only `proj_epsg` is relevant on collection asset level, I'd use the other ones only on item asset level